branding/css/setup-wizard.less

```diff
@@ -16,9 +16,9 @@ body{
       .btn-sm;
     }
     .panel{
-      box-shadow: 0px 2px 1px 0px #dadada;
-      -moz-box-shadow: 0px 2px 1px 0px #dadada;
-      -webkit-box-shadow: 0px 2px 1px 0px #dadada;
+      box-shadow: 0px 2px 1px 0px @box-shadow;
+      -moz-box-shadow: 0px 2px 1px 0px @box-shadow;
```

Comment:
Here and elsewhere, can't we automatically prefix based on a browserslist at build time?